LavalinkServer/src/main/java/lavalink/server/io/StatsCollector.kt

```diff
@@ -40,34 +40,91 @@ class StatsCollector(val socketServer: SocketServer) {
         private val os get() = si.operatingSystem
 
         private var prevTicks: LongArray? = null
+        private val ticksLock = Any()
+
+        private val CPU_STATS_REFRESH_INTERVAL_MS = 30000
```

Comment:
```suggestion
        private const val CPU_STATS_REFRESH_INTERVAL_MS = 30000
```

also not sure why `si`,`hal`,`os`, `prevTicks` & `ticksLock` were in the compation object to begin with
I think we can move those out? git blame prob goes back to me lmao